app/code/Magento/ImportExport/Block/Adminhtml/Export/Filter.php

```diff
@@ -40,18 +43,34 @@ class Filter extends \Magento\Backend\Block\Widget\Grid\Extended
         'updated_at' => \Magento\ImportExport\Model\Export::FILTER_TYPE_DATE,
     ];
 
+    /**
+     * @var \Magento\ImportExport\Model\Export\ConfigInterface
+     */
+    private $exportConfig;
+
+    /**
+     * @var \Magento\Framework\ObjectManagerInterface
+     */
+    private $objectManager;
+
     /**
      * @param \Magento\Backend\Block\Template\Context $context
      * @param \Magento\Backend\Helper\Data $backendHelper
      * @param \Magento\ImportExport\Helper\Data $importExportData
      * @param array $data
+     * @param ConfigInterface|null $exportConfig
+     * @param Magento\Framework\ObjectManagerInterface $objectManager
      */
     public function __construct(
         \Magento\Backend\Block\Template\Context $context,
         \Magento\Backend\Helper\Data $backendHelper,
         \Magento\ImportExport\Helper\Data $importExportData,
-        array $data = []
+        array $data = [],
+        ConfigInterface $exportConfig = null,
+        ObjectManagerInterface $objectManager
     ) {
+        $this->objectManager = $objectManager;
+        $this->exportConfig = $exportConfig ?: $this->objectManager->get(ConfigInterface::class);
```

Comment:
Please take a look at the example in "Adding a constructor parameter" section of [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#php). There's no need to add ObjectManager as a constructor parameter.